src/Lykke.Common.ApiLibrary/Middleware/MiddlewareApplicationBuilderExtensions.cs

```diff
@@ -37,11 +75,13 @@ public static class MiddlewareApplicationBuilderExtensions
                 await next();
             });
 
-            app.UseMiddleware<GlobalErrorHandlerMiddleware>(createGlobalErrorResponse);
+            app.UseMiddleware<GlobalErrorHandlerMiddleware>(
+                app.ApplicationServices.GetRequiredService<ILogFactory>(),
```

Comment:
Better to cache app.ApplicationServices.GetRequiredService<ILogFactory>() in local var to avoid resolbing it second time below.